webapp/components/general/Dialog.js

```diff
@@ -25,9 +25,15 @@ const AboutContent = withStyles(styles)(({ classes }) => (
   <Paper className={classes.paper}>
     <img width="250" src={logoNetpyne} />
     <Box m={1}>
-      <Typography variant="h5" style={{ color: secondaryColor }}>
+      <Link variant="h5" style={{ display: 'block'}} href="https://github.com/MetaCell/NetPyNE-UI" target="_blank">
         NetPyNE-UI v0.7.0
-      </Typography>
+      </Link>
+      <Link variant="h5" style={{ display: 'block'}} href="https://github.com/Neurosim-lab/netpyne" target="_blank">
+        NetPyNE v0.9.6
```

Comment:
v1.0.0.2 is the current version

Reply:
These will rapidly get out of date if they're not automatically generated. Maybe pull from https://github.com/MetaCell/NetPyNE-UI/blob/master/requirements.txt?

Reply:
I see what you mean and I created a [separate issue](https://github.com/MetaCell/NetPyNE-UI/issues/378) for this